app/assets/stylesheets/v3/layout_components/forms.scss

```diff
@@ -178,14 +179,15 @@ input[type="checkbox"] + label,
     content: " ";
     display: block;
     position: absolute;
-    left: 10px;
-    top: calc(50% - 12px);
-    width: 24px;
-    min-width: 24px;
-    height: 24px;
+    left: 1rem;
```

Comment:
I don't have a strong opinion about this, but setting this to `0.6rem` would result in an even gap around the top, left, bottom sides of the checkbox. (For single line checkboxes at least)
```suggestion
    left: 0.6rem;
```

![Screen Shot 2020-07-21 at 3 35 20 PM](https://user-images.githubusercontent.com/27693833/88109487-d42cd780-cb67-11ea-8fc9-044e42e87407.png)

Reply:
I was basing the spacing on these checkboxes, I think it gives a bit more breathing room. 
![image](https://user-images.githubusercontent.com/1906696/88114328-db0c1800-cb70-11ea-9bef-1e473ef3c7ca.png)